test/make.sh

```diff
@@ -32,56 +32,73 @@ done;
 # releasing to OSS are present
 function basefiles() {
   echo "Checking for required files"
+
   test -f CONTRIBUTING.md || echo "Missing CONTRIBUTING.md"
   test -f LICENSE || echo "Missing LICENSE"
   test -f README.md || echo "Missing README.md"
+
+  if ! [ "$( test -f CONTRIBUTING.md && test -f LICENSE && test -f README.md )" ]; then
+    echo " > All Required OSS files were found"
+    echo ""
+  fi
 }
 
 # This function runs the hadolint linter on
 # every file named 'Dockerfile'
+# using git ls-files because we're essentially linting for version control
 function docker() {
-  echo "Running hadolint on Dockerfiles"
-  find . -name "Dockerfile" -exec hadolint {} \;
+  echo "Linting Dockerfiles"
+  if ! [ "$(git ls-files | grep "Dockerfile" | xargs hadolint)" ]; then
```

Comment:
So this may fail sometimes as a user may have downloaded the zip and not have a `.git` directory.  That is ok.